README.md

```diff
@@ -1,284 +1,415 @@
+
+### Install:
+
+ - Install this project with: NPM install, Create a .github/workflow folder and add linter.yml in it
+ - add .hintrc file, also add .stylelintrc.json file.
+
+### Usage:
+
+- To run the project: right click on your html environment and click on run with live server.
+
+### Run Linter tests:
+
+- To run linter tests, run the following command: npx stylelint "**/*.{css,scss}" 
+
+## Deployment
+- You can deploy this project using: Host services
+
+
+
+# 👥 Author
+
+### Adesina Abiodun Paul
+
+
+
+### GitHub: https://github.com/adesina-abiodun
+
+### LinkedIn: https://www.linkedin.com/in/adesina-abiodun-paul-b513bb143
+
+### Twitter: https://twitter.com/adesinaabiodunp
+
+
+
+# 🔭 Future Features
+
+- [ ] Animation
+- [ ] Modal pages
+
+
+# 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+
+# ⭐️ Show your support
+
+
+# 🙏 Acknowledgments
+
+- I would like to thank Microverse and my code reviewer for making me a better programmer
+
+
+
+
+📝 License
+This project is [MIT](./LICENSE) licensed.
+=======
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [My Microverse portfolio project] <a name="my-portfolio"></a>
+
+> This is my microverse portfolio project. I created using HTML and CSS.
+
+## 🛠 Built With
+
+- HTML
+- CSS
+
+
+
+## Key Features
+
+- [Content]: World Heritage List
+- [HTML] : This is a Markup Language use to achieve the project
+- [Development] : Mobile first development
+- [Responsiveness] : Responsiveness and flexibility for any screen size
+- [Linters] : This is a tool used in the project to avoide coding error and professionality
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://adesina-abiodun.github.io/Microverse-Portfolio-Project/)
+
+
+
+## 💻 Getting Started
+
+### You will need the following tools to get started with the project
+
+ 1. Installation of Git
+ 2. Installation of NPM
+ 3. A working browser
+ 4. Internet connection
+ 5. A code editor like Visual Studio Code
+ 6. Installation of Live Server plugging in Visual Studio Code
+ 7. Linters
+
+## To get a local copy up and running, follow these steps:
+
+- click on the portfolio file
+- navigate to clone and copy the link
+- Clone by http or ssh (i'll advice you clone with ssh)
+- go to your vsCode and use the git clone command to clone the project to your local space. 

```

Comment:
- [ ] Please, You have duplicated the README contents, Kindly remove unnecessary the duplicated parts to make it more professional and cleaner 👍 👍